src/main/java/sirius/biz/process/ErrorContext.java

```diff
@@ -143,6 +144,33 @@ public void perform(UnaryOperator<String> failureDescription, UnitOfWork task) {
         }
     }
 
+    /**
```

Comment:
Reimplement perform above as s.th. like
`performAndGet(failureDescription, () -> { task.execute(); return null; });`

to eliminate duplicate code... also oder methods appropriately (first impl, then inner private methods, then boilerplate (performAndGet, logException, perform)